GLOSSARY.md

```diff
@@ -45,6 +45,48 @@ Class names and proper nouns do not get translated.
 - protected (function) - защищённая (функция)
 - public (function) - общедоступная (функция)
 - rebind variable - переприсвоение переменной
-- substraction - вычитание
+- subtraction - вычитание
 - signature - сигнатура (функции)
 - tail - хвост
+
+
+### Glossary (portuguese)
+
+#### Elixir-related terms
+
+- actors - atores
+- atom - átomo
+- function clauses - cláusulas de função
+- guard - guarda
+- guard operator - operador de guarda
+- keyword list - lista de palavras
+- match operator - operador para procura de padrões
+- pin operator - operator para apontar para último valor atribuido a variável
+- pipe operator - operador pipe
+- sigils - tag de padrão
```

Comment:
Not sure about this term translation. Sigils is a hard term to explain. They are used to transform the inner text into another type of data (like `~r/[0-9]/` that creates a Regex struct) or a different string representation (like `~s(this is a string with "double" quotes, not 'single' ones)` that escape the double quotes).

I would translate to something like: `operadores para transformação de strings em outras strings ou novos tipos de dados`. It's a bit longer but is more clear.

What do you think?